cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,102 @@
+const { generateWorker } = require('../support/generateWorker');
+
 describe('Web Tables page', () => {
-  it('', () => {
-    
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  const worker = generateWorker();
+
+  it('should have navigation elements', () => {
+    const selectRow = '5 rows';
+    cy.get('.-pageInfo')
+      .should('exist');
+
+    // eslint-disable-next-line cypress/unsafe-to-chain-command
+    cy.get('select')
+      .select(selectRow)
+      .should('have.value', '5');
+  });
+
+  it('should allow to add worker', () => {
```

Comment:
I see here that you have added new worker, but you didn't check if it really was added by `should` assertion